packages/flutter/lib/src/rendering/animated_size.dart

```diff
@@ -264,7 +264,8 @@ class RenderAnimatedSize extends RenderAligningShiftedBox {
   }
 
   @override
-  Size computeDryLayout(BoxConstraints constraints) {
+  @protected
```

Comment:
Here and elsewhere: Did you mean to add `@protected` to all the overrides? We usually don't do that...

Reply:
Looks like we maybe should, though, since to my surprise the override doesn't seem to inherit it...

Reply:
I filed https://github.com/dart-lang/sdk/issues/53745 to track this